clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesServerGroup.groovy

```diff
@@ -126,6 +127,30 @@ class KubernetesServerGroup implements ServerGroup, Serializable {
     }
   }
 
+  KubernetesServerGroup(V1beta1DaemonSet daemonSet, String account, List<Event> events) {
+    this.name = daemonSet.metadata?.name
+    this.account = account
+    this.region = daemonSet.metadata?.namespace
+    this.namespace = this.region
+    this.createdTime = daemonSet.metadata?.creationTimestamp?.getMillis()
+    this.zones = [this.region] as Set
+    this.securityGroups = []
+    /**
+     * Need to check if this is required or not
+     */
+    //this.replicas = daemonSet.spec?.replicas ?: 0
```

Comment:
I think it makes sense to keep for translating here, because this way you can see how many replicas are running in your cluster from the Spinnaker UI.